test/ServerComparison.TestSites/Program.cs

```diff
@@ -53,8 +53,10 @@ public static void Main(string[] args)
                 // Check that we are not using IIS inproc before we add Kestrel.
                 builder.UseKestrel();
             }
-            
             builder.UseIISIntegration();
+#if NETCOREAPP2_2
```

Comment:
Actually, you will need to special case UseIIS like we do with UseKestrel and UseHttpSys.

Reply:
Nvrm, it will only override the server if we are inproc. We should change that behavior eventually.

Reply:
Why?

Reply:
As we aren't overloading the call to ```UseIISIntegration```, we can make ```UseIIS()``` behave the same way UseKestrel and UseHttpSys do.

Reply:
> We should change that behavior eventually.
Why?

Reply:
What's the point of adding in process server when you know for sure it would not work?

Reply:
Drop the if, we'll take care of this in the test matrix. And if we ever accidentally run the net461 scenario there's a good error message, correct?

Reply:
There is no NET461 target in Server.IIS.